client/src/components/NavBar.js

```diff
@@ -20,17 +20,22 @@ export default function NavBar() {
 	};
 
 	const handleLogout = () => {
-		if(window.confirm('You sure you want to logout?')){
+		if (window.confirm("You sure you want to logout?")) {
 			localStorage.clear();
 			history.push("/login");
 		}
 	};
 
+	const goToAccount = () => {
+		history.push("/manage-account");
```

Comment:
If we change this from "account management" to "settings", this route should be updated to `/settings`